docs/settings/linux/tasks.json

```diff
@@ -121,7 +121,7 @@
       "type": "shell",
       "label": "ibus-keyman: build",
       "command": "./build.sh",
-      "args": [ "build", "--debug" ],
+      "args": [ "build", "--debug", "--coverage" ],
```

Comment:
Is `--coverage` only applicable to `test`?

Reply:
Probably; code coverage is usually a measure of how much of the codebase is being tested, after all.

Reply:
Yes and no. You'll get code coverage only when you run tests, so yes, strictly speaking it's only applicable there. However, you also need to compile with specific flags, which is why I also added it to the `configure` and `build` actions.. I don't think it hurts to always compile with these flags during development when you build in VSCode - IMO less interrupting than trying to get the code coverage and then detecting that you'll have to clean and rebuild.